script/Ciel/Day 9/09_01_CIEL09_1.txt

```diff
@@ -396,7 +438,7 @@ Are you not wearing your glasses today?"
 [sha:39afad2b083b2f183d5c45e93ea69abd963ab5d3]{
 -- Page 32, Offset 22217.
 -- 言われて、ペタ、と自分の顔に手を置いた。
-Upon hearing her words, I put my hand on my face.
+Hearing that, I reach up and touch my face.
```

Comment:
The way the below line is phrased makes me feel like it could be more interesting.

>There is nothing there but nakedness.

Or something like that.

Reply:
That is a very strange thing to say in EN imo